operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -0,0 +1,257 @@
+local dashboard = import './utils/dashboard.jsonnet';
+local panel = import './utils/panel.jsonnet';
+local filename = 'agent-cluster-node.json';
+
+{
+  [filename]:
+    dashboard.new(name='Grafana Agent Flow / Cluster Node') +
+    dashboard.withDocsLink(
+      url='https://grafana.com/docs/agent/latest/flow/reference/cli/run/#clustered-mode-experimental',
+      desc='Clustering documentation',
+    ) +
+    dashboard.withDashboardsLink() +
+    dashboard.withUID(std.md5(filename)) +
+    dashboard.withTemplateVariablesMixin([
+      dashboard.newTemplateVariable('cluster', |||
+        label_values(agent_component_controller_running_components, cluster)
+      |||),
+      dashboard.newTemplateVariable('namespace', |||
+        label_values(agent_component_controller_running_components{cluster="$cluster"}, namespace)
+      |||),
+      dashboard.newTemplateVariable('instance', |||
+        label_values(agent_component_controller_running_components{cluster="$cluster", namespace="$namespace"}, instance)
+      |||),
+    ]) +
+    // TODO(@tpaschalis) Make the annotation optional.
+    dashboard.withAnnotations([
+      dashboard.newLokiAnnotation('Deployments', '{cluster="$cluster", container="kube-diff-logger"} | json | namespace_extracted="grafana-agent" | name_extracted=~"grafana-agent.*"', 'rgba(0, 211, 255, 1)'),
+    ]) +
+    dashboard.withPanelsMixin([
+      // Node Info row
+      (
+        panel.new('Node Info', 'row') +
+        panel.withPosition({ h: 1, w: 24, x: 0, y: 0 })
+      ),
+      // Node Info
+      (
+        panel.new('Node Info', 'table') +
+        panel.withDescription(|||
+          Information about a specific cluster node.
+        |||) +
```

Comment:
Can you add documentation to the description? i.e., describe what each of the rows mean, and when the values might be concerning. I can help come up with the description if it's not clear.

Reply:
Yeah, will do.

I've added the following, let me know how it sounds

```
          Information about a specific cluster node.

          Lamport clock time: The observed Lamport time on the specific node's clock used to provide partial ordering around gossip messages. Nodes should ideally be observing roughly the same time, meaning they are up-to-date on the cluster state. If a node is falling behind, it means that it has not recently processed the same number of messages and may have an outdated view of its peers.

          Internal cluster state observers: The number of Observer functions that are registered to run whenever the node detects a cluster change.

          Gossip health score: A health score assigned to this node by the memberlist implementation. The lower, the better.

          Gossip protocol version: The protocol version used by nodes to communicate with one another. It should match across all nodes.```

Reply:
Perfect! Make it a bullet point list though, since the description can be Markdown:

```
          Information about a specific cluster node.

          * Lamport clock time: The observed Lamport time on the specific node's clock used to provide partial ordering around gossip messages. Nodes should ideally be observing roughly the same time, meaning they are up-to-date on the cluster state. If a node is falling behind, it means that it has not recently processed the same number of messages and may have an outdated view of its peers.

          * Internal cluster state observers: The number of Observer functions that are registered to run whenever the node detects a cluster change.

          * Gossip health score: A health score assigned to this node by the memberlist implementation. The lower, the better.

          * Gossip protocol version: The protocol version used by nodes to communicate with one another. It should match across all nodes.
```